metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatFunctionalSpec.groovy

```diff
@@ -226,6 +226,7 @@ class MetacatFunctionalSpec extends Specification {
 
         when:
         api.createDatabase(catalog.name, databaseName, dto)
+        Thread.sleep(5000)
```

Comment:
In the other tests the wait time is 10 seconds delay is this delay enough for the follower read?

Reply:
yeah, it turns out 5 seconds is enough. I will adjust other test to use 5 seconds as well.